js/packages/botProtocol/botProtocol.js

```diff
@@ -2,21 +2,21 @@
  * botProtocol.js
  * contains all the tokens for surface <-> robot communication
  *
- * Bobby Martin
+ * Nugget Industries
  * 2017
  */
 
 // package dependancies
 const uuidv1 = require('uuid/v1');
 
 // ye olde ghetto JS enum
-// if I wasn't a fucking idiot I'd have started writing this whole project in
 const tokenTypes = {
     RESPONSE: 'response',
     ECHO: 'echo',
     READMAG: 'readMag',
     STARTMAGSTREAM: 'startMagStream',
```

Comment:
Is 'magstream' the magnetometer stream? That implies that magnetometer data is being sent over the network, which makes sense. Does that in turn imply control loop stuff will be done on the server station? I think it may make more sense to do that ROV-side.

Reply:
Control loop stuff will happen on the ROV. The intent with magstream is so that we can see the robot's orientation from the surface station.